docs/fxa-engineering/fxa-metrics.md

```diff
@@ -100,7 +100,7 @@ The Heka filter does some processing on the events and then writes them once-per
 The CSV files are then processed by python scripts in the `mozilla/fxa-activity-metrics` repo, running on the fxa-admin EC2 instance (managed by jrgm and jbuck):
 
 * https://github.com/mozilla/fxa-activity-metrics/
-* https://github.com/mozilla-services/cloudops-deployment/blob/master/projects/fxa/puppet/modules/fxa_admin/manifests/init.pp
+* https://github.com/mozilla-services/cloudops-deployment/blob/main/projects/fxa/puppet/modules/fxa_admin/manifests/init.pp
```

Comment:
The old URL was correct here

Reply:
But tell me, this is a closed repository, I understand correctly. I just don't have such a repository.

Reply:
Yep, I know it's private.  I'm just saying the old URL is still the correct one.